### PR TITLE
Book Dash Nov 2020 Report and update templates for remote book dash as per Nov 2020 event

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* 4th Book Dash took place from Nov 9 to 13 online that needed update of format and templates

### List of changes proposed in this PR (pull-request)

* Adding report from Nov 2020 event
* Adding templates for future remote events
* Updating folder structure to have the right files in place

### What should a reviewer concentrate their feedback on?

- [ ] Need to upload Scriberia illustrations to Zenodo: Consult with @KirstieJane 
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: workshops/book-dash/book-dash-nov20-report.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+-   **Marilina Santero (Spain)** translated our promotional materials in Spanish, submitted an abstract for PyDay Chile 2020 event to present a talk about _The Turing way_ and after the Book Dash presented the talk. See the slides on Zenodo ([https://zenodo.org/record/4300282](https://zenodo.org/record/4300282))
+-   **Veronika Cheplygina (The Netherlands)** started a chapter on project workflow and Task Management that will be hosted in the Guide for Project Design ([PR #1442](https://github.com/alan-turing-institute/the-turing-way/pull/1442)).
+-   **Carlos Martinez-Ortiz (The Netherlands)** managed community contributions to the chapters in Reproducible Research from his team members from Netherlands eScience Centre that included a chapter on [Code Review](https://the-turing-way.netlify.app/reproducible-research/reviewing.html) and [PR #912](https://github.com/alan-turing-institute/the-turing-way/pull/912)).
+-   **Paul Owoicho, Neha Moopen and Emma Karoune (UK)** worked on book’s accessibility aspects that included a chapter on maintaining consistency, a chapter on GitHub for beginners, compiling glossary terms and designing templates to facilitate future contributions.
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:107]:
```suggestion
-   **Paul Owoicho (UK), Neha Moopen (The Netherlands) and Emma Karoune (UK)** worked on book’s accessibility aspects that included a chapter on maintaining consistency, a chapter on GitHub for beginners, compiling glossary terms and designing templates to facilitate future contributions.
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+Each development session followed the [coworking techniques](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-motivation.html#techniques) that we also use for our [Online Collaboration Cafés](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html). 
+To facilitate a shared timer we used the [browser-based cuckoo clock](https://cuckoo.team/tw-bookdash).
+A shared HackMD with agenda, resources and information was set up to ensure that all participants can centrally take notes and follow different session.
+You can see the template document: https://hackmd.io/@turingway/template-bookdash-nov2020.
+
+
+#### Highlights of contributions from the event
+
+-   The ethics team members, **Ismael Kherroubi Garcia, Laura Carter and Sophia Batchelor (UK)** were joined by **Irene Komal P. (India), Asha Susan Titus (UK) and Laura Acion (Argentina)** who managed to work on multiple chapters for the Guide for Ethical Research.
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:95]:
```suggestion
-  *Contributors to the Guide for Ethical Research*: **Ismael Kherroubi Garcia, Laura Carter and Sophia Batchelor (UK)** were joined by **Irene Komal P. (India), Asha Susan Titus (UK) and Laura Acion (Argentina)** who managed to work on multiple chapters for the Guide for Ethical Research. 
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    -   Irene wrote a chapter on Data Anonymisation, currently being edited in [PR #1579](https://github.com/alan-turing-institute/the-turing-way/pull/1579).
+    -   Asha started a chapter on activism ([PR #1582](https://github.com/alan-turing-institute/the-turing-way/pull/1582), which is currently edited by Laura in this PR: [#1675](https://github.com/alan-turing-institute/the-turing-way/pull/1675). 
+-   **Laura Ación (Argentina) and Lupe Canaviri Maydana (Bolivia)** worked on multiple subchapters on Leadership in Data Science available in the Guide for Collaboration, which is being co-edited by Malvika in this PR: [#1553](https://github.com/alan-turing-institute/the-turing-way/pull/1553).
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:99]:
```suggestion
-  *Leadership in Data Science*: **Laura Ación (Argentina) and Lupe Canaviri Maydana (Bolivia)** worked on multiple subchapters on leadership available in the Guide for Collaboration, which is being co-edited by Malvika in this PR: [#1553](https://github.com/alan-turing-institute/the-turing-way/pull/1553).
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+-   **Batool Almarzouq (Saudi Arabia) and Brigitta Sipőcz (USA)** together designed a subchapter on GitHub actions that will be added to the Continuous Integration chapter in the Guide for Reproducible Research ([PR #1588](https://github.com/alan-turing-institute/the-turing-way/pull/1588) and [#1558](https://github.com/alan-turing-institute/the-turing-way/pull/1558)).
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:100]:
```suggestion
-  Continuous Integration Services: **Batool Almarzouq (Saudi Arabia) and Brigitta Sipőcz (USA)** together designed a subchapter on GitHub actions that will be added to the Continuous Integration chapter in the Guide for Reproducible Research ([PR #1588](https://github.com/alan-turing-institute/the-turing-way/pull/1588) and [#1558](https://github.com/alan-turing-institute/the-turing-way/pull/1558)). 
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+    -   Batool also managed to translate the README file in Arabic ([PR #1586](https://github.com/alan-turing-institute/the-turing-way/pull/1586)).
+    -   Brigitta also participated in several discussions around software development and managing community projects, and opened a few more PRs related to _The Turing Way_ GitHub workflow.
+-   **Samuel Guay (Canada)** started to work on different aspects of translation, set up the Transifex files for the French translation, reviewed the [README-French.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-French.md) and suggested the hypothes.is for web annotation to merge into the GitHub repository.
+-   **Marilina Santero (Spain)** translated our promotional materials in Spanish, submitted an abstract for PyDay Chile 2020 event to present a talk about _The Turing way_ and after the Book Dash presented the talk. See the slides on Zenodo ([https://zenodo.org/record/4300282](https://zenodo.org/record/4300282))
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:104]:
```suggestion
-  *Translation Process*:
    -   Batool managed to translate the README file in Arabic ([PR #1586](https://github.com/alan-turing-institute/the-turing-way/pull/1586)).
  -   **Samuel Guay (Canada)** started to work on different aspects of translation, set up the Transifex files for the French translation, reviewed the [README-French.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-French.md) and suggested the hypothes.is for web annotation to merge into the GitHub repository.
  -   **Marilina Santero (Spain)** translated our promotional materials in Spanish, submitted an abstract for PyDay Chile 2020 event to present a talk about _The Turing way_ and after the Book Dash presented the talk. See the slides on Zenodo ([https://zenodo.org/record/4300282](https://zenodo.org/record/4300282))
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+
+The fourth _The Turing Way_ Book Dash was hosted online from 9-13 November 2020 by Malvika Sharan and Kirstie Whitaker.
+
+## Overview
+
+_The Turing Way_ [Book Dash events](https://the-turing-way.netlify.app/community-handbook/bookdash.html) are a less intense version of [Book Sprints](https://en.wikipedia.org/wiki/Book_sprint), where participants collaboratively work on _The Turing Way_ book synchronously to develop new chapters and review/edit existing ones to make them more accessible, comprehensive and up-to-date.
+They also contribute to enhancing the project by improving the ways we work in the community and take lead on accomplishing different tasks or subprojects.
+
+In the past, we have organised 1-1.5 day long Book Dash events in person or partially remote.
+However, with the COVID-19 pandemic situation this year, we moved the November's Book Dash online and designed them with multiple short co-working sessions spread over 5 days for flexible participation by members in different time zones.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:11]: remove "the" before "November's"

File: workshops/book-dash/book-dash-nov20-report.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+
+We developed a format which could allow people in different time zones to participate with the same efficiency and equitable support.
+This involved creating multiple small development sessions throughout the day, adding dedicated sessions for informal social interactions, developing shared documents with all the information, providing support fund to ensure that everyone can comfortably participate and hosting a pre-event call to communicate these resources to everyone.
+
+### Application, Review and Selection process
+
+As in the past, we invited applications through an open call where interested applicants could state their goals and interest for the Book Dash.
+This application was also open for the long term members from the community who may have attended a Book Dash or Collaboration Cafes in the past to join as a helper and mentor for new contributors.
+
+Applicants were asked to think about the collaborative element of the Book Dash and state how they can engage with other participants.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:51]: think "how they can" should be "how they could"

File: workshops/book-dash/book-dash-nov20-report.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+To get a sense of the time zones these applicants come from, we asked them to choose their preferred slots during the day that they can join.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:52]: think "applicants come from" should be "applicants came from"

File: workshops/book-dash/book-dash-nov20-report.md [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+You can see a template of this document [here](https://tinyurl.com/tw-bookdash-template).
+
+We used the rubrics (as explained in the [online chapter](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-application.html#reviewing-process)) to score the applications during the review and selection process.
+
+### The book dash participants and event plan
+
+Selected from a pool of applications, 20 candidates including two mentors were invited to attend this in-person event (see their names in the group picture below).
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:59]: "in-person" - should be "online"?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 10]
```diff
@@ -0,0 +1,232 @@
...
+
+![](figures/Nov20-BookDash.png)
+_Zoom screenshot of our community share-out event on 13 November 2020 at the Book Dash. Our Book Dash participants included Laura Acion (Argentina), Emma Karoune (UK), Marilina Santero (Spain/Argentina), Irene Komal (India), Asha Titus (UK), Ismael Kherroubi Garcia (Spain/UK), Martia Mangiarulo (UK/Italy), Laura Carter (UK), Neha Moopen (The Netherlands), Paul Owoicho (UK/Nigeria), Batool Almarzouq (Saudi Arabia/UK), Samuel Guay (Canada), Martina Vila (Germany), Sarah Gibson (UK), Sophia Batchelor (UK), Carlos Martinez-Oritz (The Netherlands), Lupe Canaviri Maydana (Bolivia) and Sangram Keshari Sahu (India). Several community members joined the share-out call on the last day._
+
+A participation support fund was made available for the participants to purchase small equipment or other resources (such as headphone, webcam, second screen, or software) that can improve the quality of their participation and enhance their overall experience of working with us for the entire week.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:64]: "that can improve" -> "that could improve"?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+They were asked to select at least one development session from the following slots each day (time given in UTC):
+- Development session 1: 08:00 - 10:30
+- Development session 2: 11:00 - 13:30
+- Development session 3: 17:30 - 20:00
+
+A longer break from 13:30 - 17:30 was given to allow people to participate in informal events, self-organised activities, accomplish their day-to-day task outside the Book Dash or take a break as required.
+
+## Pre-dash drop-in calls: 4 November 2020
+
+We hosted 2 pre-dash drop-in calls on 4 November 2020 to ensure that all our participants were familiar with the project, book dash and how they can get the most out of their participation ([HackMD notes](https://hackmd.io/@turingway/bookdash-nov2020-drop-in)).
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:74]: Should Book Dash be capitalised here (not sure what the style guide says)?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+The main aims of this call were to get to know our attendees, connect them with others and address their questions before the book dash.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:75]: Should Book Dash be capitalised here (not sure what the style guide says)?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 13]
```diff
@@ -0,0 +1,232 @@
...
+
+Using a [SMART goal template](https://freespiritpublishingblog.com/wp-content/uploads/2015/08/setting-smart-goals.pdf) (by Danielle R. Schultz [2015]), we also facilitated goal-setting exercise for all the attendees to get them prepared for their participation during the Book Dash week.
+
+Several participants shared their contribution plans and topics they would like to add in _The Turing Way_ that is either missing in the existing chapters or need to be updated. 
+You can read notes from these calls [here](https://hackmd.io/@turingway/bookdash-nov2020-drop-in).
+
+## Book Dash Week: 9-13 November 2020
+
+Each day we will host 3 development sessions and one open slot where participants could plan their time to interact with others, continue working or take a break as per their need.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:84]: "each day we will host" -> "each day we hosted" 

Also this line seems a repeat of the event plan above - consider deleting?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 14]
```diff
@@ -0,0 +1,232 @@
...
+
+All the participants were very supportive of each other's work and reviewed PRs beyond their contributions.
+Two core contributors **Martina G. Vilas and Sarah Gibson** also participated in the Book Dash to facilitate mentored contributions, especially by supporting participants who were new to the projects.
+Additionally, Sarah reviewed several Pull Requests and helped Malvika merge the chapter on [Remote Collaboration](https://the-turing-way.netlify.app/collaboration/remote-collab.html).
+Martina onboarded Lupe and Laura by offering mentorship in Spanish and helped Batool integrate [web.hypothes.is](https://web.hypothes.is/) to enable [web-annotation and commenting](https://jupyterbook.org/interactive/comments.html) on _The Turing Way_ content.
+On 12 November 2020, Martina also gave a talk at the Brainhack Donostia 2020 on Computational Reproducibility. 
+Under the title “A How-To Guide Based on The Turing Way”, she shared useful tips and tools for computational reproducibility. Her talk is also available on Zenodo: [http://doi.org/10.5281/zenodo.4269795](http://doi.org/10.5281/zenodo.4269795).
+
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:125]: Such an awesome list of what happened during the week!

File: workshops/book-dash/book-dash-nov20-report.md [hunk 15]
```diff
@@ -0,0 +1,232 @@
...
+Particularly, on Day-2 we hosted a themed Coffee discussion with the topic "Science and Open source role models" where attendees shared their personal stories of individuals who inspired them to pursue science or open source practices.
+On Day-3 we hosted the "Show and Tell - Family Meal" for people to get to know each other over a meal (breakfast, lunch or dinner based on their time zone).
+This was done to mainly emulate how we would have shared meals and enjoyed special treats if the event were to take place in person.
+
+### Community Share-out
+
+On the last day, we hosted only 2 development sessions to document the current status and next steps of the projects started by attendees during the Book Dash.
+
+Each session was followed by a 2-hour long community share-out events that were open for the community members to learn about what our Book Dash attendees have worked on.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:147]: "attendees have worked on" -> "attendees had worked on"

File: workshops/book-dash/book-dash-nov20-report.md [hunk 16]
```diff
@@ -0,0 +1,232 @@
...
+Each participant got to demonstrate their work through small presentation and celebrate the collaborative work they got to do together. 
+You can watch recordings from these share-out sessions in [this YouTube playlist](https://www.youtube.com/watch?v=EZl6vgPzlvg&list=PLBxcQEfGu3DlXmOGBN8UKzFYITqKrgCwc&index=2).
+Read more about the contributors in [issue #1584](https://github.com/alan-turing-institute/the-turing-way/issues/1584).
+
+## Illustrations by Scriberia Artist: Jem Milton
+
+Illustration by Scriberia artists had added enormously to the productivity, enthusiasm and outcome of the Boo Dash events in the past.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:154]: "Boo Dash" - "Book Dash"

File: workshops/book-dash/book-dash-nov20-report.md [hunk 17]
```diff
@@ -0,0 +1,232 @@
...
+We continued this tradition by inviting Jem Milton this time, who spent 3 days, from 10 to 12 November, interacting with the participants and developing new sets of illustration for _The Turing Way_ community. 
+Our attendees discussed their ideas with Jem and captured important concepts they were working on during the book dash or something that they are interested to work on in the future.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:156]: Should Book Dash be capitalised here (not sure what the style guide says)?

File: workshops/book-dash/book-dash-nov20-report.md [hunk 18]
```diff
@@ -0,0 +1,232 @@
...
+
+- The learning curve when it comes to getting across the GitHub is a little steep. 
+- I think having other people responsible for answering some of the questions will be great and even help to moderate some of the session.
+- I like the HackMD, but I found it a bit difficult to navigate as it got bigger and bigger.
+- An optional training session on GitHub or a parallel activity just for beginners could be useful.
+- Sometimes the Pomodoro doesn't work perfectly with that internal work timing.
+
+## What's next?
+
+Our global community of contributors like these book dash attendees have been helping us develop a truly community-led guidebook by including their perspectives and voice into the project.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/book-dash-nov20-report.md:195]: Should Book Dash be capitalised here (not sure what the style guide says)?

**Review by Arielle-Bennett — COMMENTED** [review]
Reviews Book Dash Nov 2020 Report - minor typoes, but otherwise looking great!

File: workshops/book-dash/book-dash-nov20-report.md [hunk 19]
```diff
@@ -0,0 +1,232 @@
...
+We developed a format which could allow people in different time zones to participate with the same efficiency and equitable support.
+This involved creating multiple small development sessions throughout the day, adding dedicated sessions for informal social interactions, developing shared documents with all the information, providing support fund to ensure that everyone can comfortably participate and hosting a pre-event call to communicate these resources to everyone.
+
+### Application, Review and Selection process
+
+As in the past, we invited applications through an open call where interested applicants could state their goals and interest for the Book Dash.
+This application was also open for the long term members from the community who may have attended a Book Dash or Collaboration Cafes in the past to join as a helper and mentor for new contributors.
+
+Applicants were asked to think about the collaborative element of the Book Dash and state how they could engage with other participants.
+To get a sense of the time zones these applicants come from, we asked them to choose their preferred slots during the day that they can join.
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:52]:
```suggestion
To get a sense of the time zones these applicants came from, we asked them to choose their preferred slots during the day that they can join.
```

File: workshops/book-dash/book-dash-nov20-report.md [hunk 20]
```diff
@@ -0,0 +1,232 @@
...
+You can see a template of this document [here](https://tinyurl.com/tw-bookdash-template).
+
+We used the rubrics (as explained in the [online chapter](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-application.html#reviewing-process)) to score the applications during the review and selection process.
+
+### The book dash participants and event plan
+
+Selected from a pool of applications, 20 candidates including two mentors were invited to attend this in-person event (see their names in the group picture below).
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:59]:
```suggestion
Selected from a pool of applications, 20 candidates including two mentors were invited to attend this online event (see their names in the group picture below).
```

File: workshops/book-dash/book-dash-nov20-report.md (hunk 16 above)
Comment (malvikasharan) [reply workshops/book-dash/book-dash-nov20-report.md:153]: This one! 😅

File: workshops/book-dash/book-dash-nov20-report.md [hunk 21]
```diff
@@ -0,0 +1,231 @@
...
+Each session was followed by a 2-hour long community share-out events that were open for the community members to learn about what our Book Dash attendees had worked on.
+Each participant got to demonstrate their work through small presentation and celebrate the collaborative work they got to do together. 
+You can watch recordings from these share-out sessions in [this YouTube playlist](https://www.youtube.com/watch?v=EZl6vgPzlvg&list=PLBxcQEfGu3DlXmOGBN8UKzFYITqKrgCwc&index=2).
+Read more about the contributors in [issue #1584](https://github.com/alan-turing-institute/the-turing-way/issues/1584).
+
+## Illustrations by Scriberia Artist: Jem Milton
+
+Illustration by Scriberia artists had added enormously to the productivity, enthusiasm and outcome of the Book Dash events in the past.
+We continued this tradition by inviting Jem Milton this time, who spent 3 days, from 10 to 12 November, interacting with the participants and developing new sets of illustration for _The Turing Way_ community. 
+Our attendees discussed their ideas with Jem and captured important concepts they were working on during the book dash or something that they are interested to work on in the future.
```
Comment (malvikasharan) [inline workshops/book-dash/book-dash-nov20-report.md:155]:
```suggestion
Our attendees discussed their ideas with Jem and captured important concepts they were working on during the Book Dash or something that they are interested to work on in the future.
```

**Comment by malvikasharan** [comment]
@Arielle-Bennett Thank you so much for the thorough review. I have integrated all your suggestions.
Can you please approve this to merge?

**Comment by ArielleBL** [comment]
Approving this :)

**Review by ArielleBL — APPROVED** [review]

**Review by Arielle-Bennett — APPROVED** [review]

**Pull request merged**